### PR TITLE
Change item/xp merge and entity tracking range defaults

### PR DESCRIPTION
**Pull request by kennytv**
Restore vanilla defaults for the merge radius, increase the tracking range values as the current ones are absolute ass. I also increased `raiderActivationRange`, as raiders like to get stuck just out of reach and become unfindable.

Waiting on feedback on possibly changing other config values. No Paper comments because it's just a Spigot file and Paper go zoom

### CODE REVIEW

File: patches/server/0005-Paper-config-files.patch [hunk 1]
```diff
@@ -5111,6 +5111,47 @@ index a6d2ce801b236b046b94913bccf7eccfc561f35a..b244b65799d9c082b8b1639bad157274
...
+     }
+ 
+-    public int playerTrackingRange = 48;
+-    public int animalTrackingRange = 48;
+-    public int monsterTrackingRange = 48;
+-    public int miscTrackingRange = 32;
++    public int playerTrackingRange = 128;
++    public int animalTrackingRange = 96;
++    public int monsterTrackingRange = 96;
++    public int miscTrackingRange = 96;
```
Comment (masmc05) [inline patches/server/0005-Paper-config-files.patch:5151]: Why can't tracking ranges be -1 by default, which will result in using vanilla values?

Vanilla has various values for each mob, setting a universal value per category already will result in behaviour differences without any possibility to restore it

**Comment by EterNityCH** [comment]
Overall a step in the right direction as most of the config values are outdated for modern Minecraft. Considering majority of the server are small, this will restore some Vanilla experience and addresses some common question people asked (like cannot see players too far out).

File: patches/server/0005-Paper-config-files.patch (hunk 1 above)
Comment (kennytv) [reply patches/server/0005-Paper-config-files.patch:5151]: The purpose of this pr is to improve defaults, not to necessarily make them equal to vanilla, and those are visual only anyways

**Comment by Leguan16** [comment]
Does this need changes in the vanilla breaking changes docs pr?

**Comment by kennytv** [comment]
Yess, you can remove the entries for the two merge radius values

File: patches/server/0005-Paper-config-files.patch [hunk 2]
```diff
@@ -5111,6 +5111,47 @@ index a6d2ce801b236b046b94913bccf7eccfc561f35a..b244b65799d9c082b8b1639bad157274
...
+-        this.itemMerge = this.getDouble("merge-radius.item", 2.5 );
++        this.itemMerge = this.getDouble("merge-radius.item", 0.5 );
+         this.log( "Item Merge Radius: " + this.itemMerge );
+     }
+ 
+     public double expMerge;
+     private void expMerge()
+     {
+-        this.expMerge = this.getDouble("merge-radius.exp", 3.0 );
++        this.expMerge = this.getDouble("merge-radius.exp", 0.5 );
```
Comment (Machine-Maker) [inline patches/server/0005-Paper-config-files.patch]: It was pointed out that the vanilla value for this is actually to disable it completely with `-1`. There are 2 exp merge systems in the server, this only controls the custom one spigot added, and doesn't mess with vanilla's own merge system. So disabling this completely will result in vanilla xp behavior.

Comment (EterNityCH) [reply patches/server/0005-Paper-config-files.patch]: Oh I was not aware of this and yeah that would certainly be a more favorable result.

**Comment by EterNityCH** [comment]
The value is perfectly ok for small SMP which is what majority of server owners are according to bstat, the value does not affect existing servers, not to mention, it can also be changed at owners discretion.

**Pull request merged**